modules/build/src/main/scala/scala/build/Inputs.scala

```diff
@@ -127,6 +127,8 @@ object Inputs {
 
   sealed trait SingleElement extends Element
 
+  sealed trait AnyScript
```

Comment:
NIT Make that extend `Element`, so that we know it's related to it.
```suggestion
  sealed trait AnyScript extends Element
```